src/main/java/io/gatling/demostore/security/SecurityConfig.java

```diff
@@ -26,31 +26,29 @@ public PasswordEncoder encoder() {
     @Override
     protected void configure(AuthenticationManagerBuilder auth) throws Exception {
         auth
-            .userDetailsService(userDetailsService)
-            .passwordEncoder(encoder());
-
+                .userDetailsService(userDetailsService)
+                .passwordEncoder(encoder());
```

Comment:
IMHO, we don't want the `BCryptPasswordEncoder` (see line 23). We don't want to burn CPU for the sake of computing a crypto-safe hash. On the contrary, we want the easiest way to provide credentials to play with. Use a `NoOpPasswordEncoder`.